### ISSUE TITLE
Add assessment weeks into database

### ISSUE THREAD

**Issue by medensqui** [PlanHubMe/PlanHub#315]
For you Middle Schoolers,

Every week, only certain classes can give tests.

We should add this in the database. I can if you guys want me to

**Comment by thatoddmailbox** [comment]
@medensqui I guess this would go up at the top of the planner week (where days off are normally recorded)? Maybe we could add a separate setting so that Middle School doesn't have to see them...

**Comment by medensqui** [comment]
@thatoddmailbox Yes. We could also add like a "when u sign in" (IF UR GRADE IS ≥ 4 and ≤ 8) THEN HIDE THEM

**Comment by thatoddmailbox** [comment]
@medensqui you wouldn't want to check that only on sign up--otherwise, this year's 8th graders won't see the high school stuff next year.

**Comment by medensqui** [comment]
@thatoddmailbox Tru

**Comment by willbarkoff** [comment]
Problem:

> Every week, only certain classes can give tests.
> -@medensqui

This is false.

**Comment by thatoddmailbox** [comment]
@nywillb Not in high school it isn't!

This is how Mondays in Feburary look for us:
![image](https://cloud.githubusercontent.com/assets/1904587/12423945/d9b2800c-be9b-11e5-8071-1118b82a8803.png)

**Comment by willbarkoff** [comment]
OHHH

**Comment by willbarkoff** [comment]
Sorry I misread the comment

**Comment by thatoddmailbox** [comment]
@medensqui Do you want this to be with the other planner announcements (at the top, saying things like "Professional Development Day" or whatever) or as something else? It probably should be possible to disable it for middle schoolers...
